### PR TITLE
Makes priming a grenade logged in combat instead of game

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

image

## Why It's Good For The Game

If you ignite a grenade and don't throw it, it isn't logged in the individual logs, so admins have to go check bomber logs instead. This is very confusing since bombs are part of combat so using one should show up in your individual logs' attack logs.

## Changelog

:cl:
admin: Priming grenades are now logged in individual's attack logs.
/:cl:

### CODE REVIEW

**Comment by rkvothe14** [comment]
image

HAS HAS!

File: code/game/objects/items/grenades/grenade.dm [hunk 1]
```diff
@@ -102,6 +102,7 @@
 
 /obj/item/grenade/proc/log_grenade(mob/user)
 	log_bomber(user, "has primed a", src, "for detonation")
+	log_combat(user, src, "has primed a", src, "for detonation")
```
Comment (ShizCalev) [inline code/game/objects/items/grenades/grenade.dm:105]: Should just have log bomber handle this for everything

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/game/objects/items/grenades/grenade.dm (hunk 1 above)
Comment (JohnFulpWillard) [reply code/game/objects/items/grenades/grenade.dm:105]: log bomber doesn't appear in individual player logs in-game, which is what I was hoping to solve with this.

Comment (ShizCalev) [reply code/game/objects/items/grenades/grenade.dm:105]: i meant add a log_combat line to the log_bomber proc so it's just automatic for everything that uses it.

Comment (ShizCalev) [reply code/game/objects/items/grenades/grenade.dm:105]: image
you're fixing the wrong thing in that case

Comment (ShizCalev) [reply code/game/objects/items/grenades/grenade.dm:105]: this should be going to LOG_ATTACK instead of LOG_GAME

Comment (JohnFulpWillard) [reply code/game/objects/items/grenades/grenade.dm]: You're right
This is what it looks like now.

image

**Pull request merged**